transcom/mymove
# B20446 int kb1 payment requests doc viewer

## Agility ticket
[B20446](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20446)

## Summary
The change was to display "No documents provided" in the document view in when reviewing a payment request with no documents.  

### How to test
* Customer: Create a move with an HHG shipment.
* SC: Approve move.
* TOO: Approve move.
* Prime:
  - Update shipment destination
  - Update shipment dates
    * Schedule Pickup Date
    * Actual Pickup Date
    * Schedule Delivery Date
  - Update shipment weights
    * Estimate Weight
    * Actual Weight
  - Create payment request with no documents
    * Create payment request
    * Do NOT upload any docs for the payment request
  - Create payment request with documents
    * Create payment request
    * Upload document to the payment request
* TIO
  - 1st payment request
    * Click "Review service items" button
    * Confirm "No documents provided" is displayed in the document viewer. 
  - 2nd payment request
    * Click "Review service items" button
    * Confirm a document is displayed in the document viewer. 

## ANDI Screenshots
![image](https://github.com/user-attachments/assets/e205f9b3-e64f-497a-aa27-e020ed7e8ab9)

![image](https://github.com/user-attachments/assets/1e1010f5-88d5-41e6-964a-1562cb106ae9)
"No documents provided" has no action item so there's nothing to do for it.

## Testing Screenshots
none
